utils/utils.go

```diff
@@ -44,20 +44,26 @@ func WalkFiles(path string, fn func(path string, info os.FileInfo)) {
 		done <- struct{}{}
 	}()
 	err := filepath.Walk(path, func(path string, info os.FileInfo, err error) error {
-		if err != nil {
-			log.Println("While traversing those files, I got this error:")
-			log.Println(err)
-			log.Println("while looking at this path:")
-			log.Println(path)
-			return err
-		}
 		if config.ExcludeFromBackup(path) {
+			if info == nil {
```

Comment:
I wonder if this should be `if info == nil || err != nil {`? I don't know for sure, just a thought.

Reply:
[The docs](https://golang.org/pkg/path/filepath/#WalkFunc) say those are equivalent, if I'm reading them right. I would think in the case where we have a valid FileInfo object we can decide to skip the whole directory (because we will be able to tell if it is a directory).

Reply:
OK